### PR TITLE
Formatting & linting

### PR DESCRIPTION
**Pull request by TristanWYL**

### CODE REVIEW

File: src/utils/processes.ts [hunk 1]
```diff
@@ -3,66 +3,68 @@ import { invoke } from "@tauri-apps/api/tauri";
...
+export const terminateProcess = (pid: number) => {
+  return invoke("terminate_process", { pid }) as Promise<void>;
 };
 
 // Cross platform function that returns an array of objects of type ProcessInfo.
 // For each platform the processing of data and mapping of properties is implemented differently.
-export async function getProcesses(): Promise<ProcessInfo[]> {
-  if (isMac) {
+export const getProcesses = async (): Promise<ProcessInfo[]> => {
+  if (await isMac) {
```
[the remark this answers is not in the document]
Comment (TristanWYL) [reply src/utils/processes.ts:15]: here isMac is a Promise. I have updated the definition of isMac.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,9 @@
     "build": "vite build",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "tauri": "tauri"
+    "tauri": "tauri",
+    "lint": "npx prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
+    "format": "npx prettier --ignore-path .gitignore --write --plugin-search-dir=. ."
```
Comment (petukhov) [inline package.json:13]: i think this is using global prettier installation. should be using local one from the package.json instead.

File: package.json [hunk 3]
```diff
@@ -8,7 +8,9 @@
     "build": "vite build",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "tauri": "tauri"
+    "tauri": "tauri",
+    "lint": "npx prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
```
Comment (petukhov) [inline package.json:12]: formatting shouldn't be inside the "lint" command, because these have different purposes.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,7525 @@
+{
```
Comment (petukhov) [inline package-lock.json:1]: don't we already have yarn.lock? why add package-lock.json?

Comment (TristanWYL) [reply package-lock.json:1]: This is not added intentionally. Maybe it is added when I installed some libraries.

**Review by petukhov — APPROVED** [review]

**Pull request merged**